# All of the information for a controller
# index 0 of motors should be channel 1 and index 1 channel 2

Header header
RoboteqMotorInfo[] motors